packages/components/src/models.ts

```diff
@@ -408,7 +408,14 @@ export class EditorModel
             columns.forEach((col, cn) => {
                 const values = this.getValue(cn, rn);
 
-                if (col.isLookup()) {
+                let renderer;
+                if (col.columnRenderer) {
+                    renderer = getQueryColumnRenderers().get(col.columnRenderer.toLowerCase());
+                }
+
+                if (renderer?.getEditableRawValue) {
+                    row.set(col.name, renderer.getEditableRawValue(values));
```

Comment:
This isn't doing anything. It would need to use `row = row.set(...)` to mutate the map. That said, can it possibly just be reverted entirely? Was this necessary just for the `alias` column?

Reply:
Yes that is just for the Alias column and is needed.  It was a bug I fixed that it wasn't setting the value back to row.  Instead of putting the special handling directly in here I thought it better to put in their renderer. I just added a comment if that helps clarify for next developer